chart/values.yaml

```diff
@@ -15,6 +15,9 @@ nameOverride: "flanksource-ui"
 fullnameOverride: "flanksource-ui"
 oryKratosURI: ""
 
+# Option to disable auth on frontend.
+withoutOrySession: true
```

Comment:
Can we rename this to just authentication?

Reply:
change to `withoutAuth`.